apps/server/src/app.ts

```diff
@@ -118,6 +119,7 @@ const checkStart = (currentState: OntimeStartOrder) => {
 export const startDb = async () => {
   checkStart(OntimeStartOrder.InitDB);
   await dbLoadingProcess;
+  populateStylesFile;
```

Comment:
does the function not need to be called?

I am also conflicted about the idea of using a function called `startDb` to do this.
Perhaps we change the function name to `init`?

Reply:
im unsure about why this works ... but it works 😅. I guess it is because we are exporting a const in loadStyle?

Reply:
we could move this call to the startServer function. it doesen't really matter when in the start process it get's called

Reply:
That is right, that seems to be a copy-paste artifact.
populateStylesFile is the value of the execution of populateStyles()
`export const populateStylesFile = populateStyles();`

This was necessary in the dbLoading because of the await, but not here.
Would it be more explicit to call populateStyles function directly?

When it comes to moving it, that is an excellent question.
The express server needs the files ready when it spins up. so I am surprised you haven't found any issues.
This populating function should run at the top of the app

Reply:
the express server get's initialized in the startServer function wich runs after startDB. So maybe we could populate the styles file at thhe top of that function?

Reply:
thats right. I would then suggest that we continue using the startDb function but rename it to init or initAssets, we would also need to rename the step in the enum